.github/workflows/retag-dev-to-test.yml

```diff
@@ -40,82 +34,36 @@ jobs:
         with:
           github-token: ${{ github.token }}
           ms-teams-webhook-uri: ${{ env.MS_TEAMS_WEBHOOK_BUILD_CHANNEL }}
-          notification-summary: PIMS CI-CD GitHub Action STARTED in TEST environment
+          notification-summary: PIMS Release DEV to TST Started
           notification-color: 17a2b8
           timezone: America/Los_Angeles
 
-  build-frontend:
+  retag:
     name: Build frontend
     needs: ci-cd-start-notification
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Source Code
         uses: actions/checkout@v3
```

Comment:
This step can now be removed - we don't need to checkout the source from git to run **oc** commands